### PR TITLE
[WIP] New opcodes

### PR DESCRIPTION
**Pull request by husho**
New pull request, because I broke old one with branch fuckery

Implemented new opcodes, improved documentation, minor changes, fixes

### CODE REVIEW

File: rwengine/src/script/modules/GTA3ModuleImpl.inl [hunk 1]
```diff
@@ -7949,16 +7949,17 @@ void opcode_02dd(const ScriptArguments& args, const ScriptString areaName, Scrip
...
     	// Create a list of candidate characters by iterating and checking if the char is in this zone
     	std::vector<std::pair<GameObjectID, GameObject*>> candidates;
-    	for(auto& p : args.getWorld()->pedestrianPool.objects) {
+    	for (auto& p : args.getWorld()->pedestrianPool.objects) {
     		auto character = static_cast<CharacterObject*>(p.second);
 
     		// We only consider characters walking around normally
     		/// @todo not sure if we are able to grab script objects or players too
-    		if(character->getLifetime() != GameObject::TrafficLifetime) {
+            // husho: only grab traffic objects
```
Comment (husho) [inline rwengine/src/script/modules/GTA3ModuleImpl.inl]: seems like whitespace is broken in this file, its either tabs or spaces

**Comment by ShFil119** [comment]
I'm not a man with power to merge, but can you divide this pull into smaller ones?
- it's hard to read and review code,
- unknown progress

**Comment by husho** [comment]
I guess it's hard to read, because all opcodes are crammed into one big file. So they should be separated into sets of 50/100.

**Comment by ShFil119** [comment]
I wasn't thinking about nr of code lines, but grouping opcodes if they are touching the same aspect of engine. ;)

**Comment by husho** [comment]
Well, this is harder than I expected, because some commits contain changes from different, as you said, aspects of the engine. Don't really know what to do now.

**Pull request closed without merging**